data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -153,11 +142,36 @@ $IPTABLES -t raw -A DHCP_IRONIC -j DROP
 {{end}}
 
 # Wait for images to be downloaded/ready
-podman wait -i 1000ms ipa-downloader
+#podman wait -i 1000ms ipa-downloader
 podman wait -i 1000ms coreos-downloader
-while ! curl --fail http://localhost/images/rhcos-ootpa-latest.qcow2.md5sum ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.initramfs ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.kernel ; do sleep 1; done
+
+# Embed agent ignition into the rhcos live iso
+sudo podman run -d --net host --privileged --name configure-coreos \
+     --restart on-failure \
+     --env IRONIC_RAMDISK_SSH_KEY="$IRONIC_RAMDISK_SSH_KEY" \
+     --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
+     --entrypoint /bin/configure-coreos-ipa \
+     -v $AUTH_DIR:/auth:ro \
+     -v $IRONIC_SHARED_VOLUME:/shared:z ${IRONIC_IMAGE}
+
+# Run httpd here to populate the IRONIC_KERNEL_PARAMS
+# Note: the IRONIC_HTTPD while set to false, will configure the KERNEL params
+# but not start the httpd server.
+podman run -d --net host --privileged --name httpd \
+     --restart on-failure \
+     --env IRONIC_RAMDISK_SSH_KEY="$IRONIC_RAMDISK_SSH_KEY" \
+     --env PROVISIONING_INTERFACE=$PROVISIONING_NIC \
+     --env IRONIC_HTTPD=true \
```

Comment:
this should be needed only if the entrypoint is ironic-api
/bin/runhttpd will always start the httpd server

Reply:
+1. Sorry I got things mixed up.